compiler/Cargo.lock

```diff
@@ -175,16 +175,16 @@ dependencies = [
 
 [[package]]
 name = "clap"
-version = "3.1.18"
+version = "3.2.8"
 source = "registry+https://github.com/rust-lang/crates.io-index"
```

Comment:
The noise here is because automatic updates https://github.com/facebook/relay/commit/8e25d86ab184d0f656a075fc4678dec4c42727b6, https://github.com/facebook/relay/commit/3cfc85ff0b89440361f91ad26406768cdb76cf12, https://github.com/facebook/relay/commit/89357466f26735f7c5d1798f9ad4b1ed0a14e056 etc aren't also updating the lock file.

It would be nice if they were either kept in sync, or have this file ignored